src/NSwag.CodeGeneration.TypeScript/Templates/FetchClient.liquid

```diff
@@ -51,6 +51,12 @@
 {%-    if UseAbortSignal -%}
             signal,
 {%-    endif -%}
+{%-       if WithCredentials -%}
+            credentials: 'include',
+{%-        endif -%}
+{%-       if UseCorsMode -%}
+            mode: 'cors',
```

Comment:
because there are three possible values: https://developer.mozilla.org/en-US/docs/Web/API/RequestInit#mode should this be an enum on settings side? I don't know about other clients if they support same options, but then it would be `public enum CorsMode` or `FetchCorsMode` with default of `None`, client template model could have `Mode` property which would map enum to string version

Reply:
Yeah, I can change to be an enum that I assume would be nullable so people don't have to have it set?

Reply:
Any objection to me using the following enum?

https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.webassembly.http.browserrequestmode?view=aspnetcore-9.0

Reply:
Short of me just creating a boolean for each different enum option in NSwag.CodeGeneration.TypeScript\Models\TypeScriptClientTemplateModel.cs. Which feels hacky but would be relatively easy.

Reply:
Just add a string property that returns correct value, null should evaluate to false in bif check